src/api/v3/feedback/__init__.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+from django.http import HttpResponse
+
+
+def feedback_handler(func):
+    def handle_feedback(request, region_slug, language_code):
+        if request.method != 'POST':
+            return HttpResponse('Invalid request.', status=405)
```

Comment:
The wordpress api answered `JsonResponse` here, so it's probably a good idea to keep it that way (also involves all other api responses)...

Reply:
So, `return JsonResponse('Invalid request.', status=405, safe=False` ?

Reply:
In the page feedback endpoint, HttpResponses are returned. Is that an error there?

Reply:
The data should be passed as dict, so `return JsonResponse(['Invalid request.'], status=405, safe=False)`I think (but I didn't try it out)...
The page feedback endpoint was implemented during the expedia hackathon, where we didn't have much time for in-depth code reviews. It turns out, that we missed some error there.

Reply:
Sorry, `['Invalid request.']` is of course a list and not a dict :man_facepalming: 
So, it would something like `{'error': 'Invalid request.'}`